students/AndrewMiotke/lesson01/assignment/inventory_management/electric_appliance_class.py

```diff
@@ -0,0 +1,37 @@
+""" Electric appliances class """
```

Comment:
One of the requirements for pylint is to add the __init__.py file under inventory_management. Don't forget to add that!

Reply:
Don't forget to get your pylint rating to meet the successful grade

Reply:
Added the __init__.py file. Thanks Ron.